exec/cnex/gc.h

```diff
@@ -0,0 +1,19 @@
+#ifndef GC_H
+#define GC_H
+#include <stdint.h>
```

Comment:
I'm not entirely sure why, but on my system the `stdint.h` header does not declare `size_t`. But `stddef.h` for example does. The documentation says a bunch of choices *should* work: https://en.cppreference.com/w/c/types/size_t

Since you need `stdint.h` for `uint64_t`, I suggest including both.

Reply:
Fixed.